astroplan/tests/test_core.py

```diff
@@ -824,3 +804,7 @@ def test_Observer_altaz(self):
         with self.assertRaises(TypeError):
             obs = Observer(location=EarthLocation(0, 0, 0))
             _ = obs.altaz(Time('2000-01-01 00:00:00'), ['00:00:00','00:00:00'])
+
+    def test_bad_site(self):
+        with self.assertRaises(KeyError):
+            _ = get_site('nonexistent site')
```

Comment:
Add newline at end of file.
What's the point in assigning to `_ =`? Can't you just remove that?
(if yes, do it everywhere)